research/deep_speech/data/vocabulary.txt

```diff
@@ -0,0 +1,33 @@
+# List of alphabets (utf-8 encoded). Note that '#' starts a comment line, which
+# will be ignored by the parser.
+# begin of vocabulary
+a
+b
+c
+d
+e
+f
+g
+h
+i
+j
+k
+l
+m
+n
+o
+p
+q
+r
+s
+t
+u
+v
+w
+x
+y
+z
+'
+ 
+-
```

Comment:
This is such a short vocab file; is it easier to just create a list in Python?

```
import string
vocab = list(string.ascii_lowercase) + ["'", " ", "-"]
```

or something to that effect?

Reply:
I think using a python list is fine for our purpose now. However, it seems like a common practice to put those vocabulary into a standalone file and let python code read from it to construct a dictionary. I guess it's because for some languages such as Chinese the character set is just too huge. At this point I prefer still using the txt file so that we can better separate code and data.

Reply:
Leaving it as a file is fine in that case-- thanks